Mathlib/Combinatorics/Additive/Energy.lean

```diff
@@ -14,62 +16,77 @@ import Mathlib.Data.Fintype.Prod
 This file defines the additive energy of two finsets of a group. This is a central quantity in
 additive combinatorics.
 
+## Main declarations
+
+* `Finset.addEnergy`: The additive energy of two finsets in an additive group.
+* `Finset.mulEnergy`: The multiplicative energy of two finsets in a group.
+
+## Notation
+
+The following notations are defined in the `AddCombi` scope:
```

Comment:
Is there precedent for `AddCombi`? Given this is written just once in a file, I'd suggest the longer `AdditiveCombinatorics`, though `mulEnergy` suggests that `PointwiseCombinatorics` might be more apt.

Reply:
This is the first use. There will probably be more uses, though

Reply:
In that case I think you should at least spell out `Combinatorics` in full. How about `open scoped Combinatorics.Pointwise`?

Reply:
`Pointwise` is really misguided here. Additive combinatorics is wider than just the study of sumsets, and sumsets do not only show up in additive combinatorics, even within combinatorics.

Reply:
In that case, perhaps `open scoped Combinatorics.Additive`?

Reply:
"Arithmetic combinatorics" is a term that's sometimes used to include both addition and multiplication, but personally I think "Additive combinatorics" is plenty clear

Reply:
> In that case, perhaps `open scoped Combinatorics.Additive`?

Yes that's what I renamed it to